### PR TITLE
New mutation

### PR DESCRIPTION
**Pull request by coleanthony1990**
# Pull Request

## Description:

- added tests for single results page
- change queries to accept coordinates and price
- add map view of favorited biz
- add eat local icon for bad images
- lots o styling

### CODE REVIEW

File: cypress/e2e/favorites-spec.cy.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ describe('Favorites Page', () => {
     cy.get('[data-cy="fav-user-message"]').contains(`You don't have any favorites yet`);
   })
   
-  it('a user should be able to add a favorite and search through favorites', () => {
+  it.only('a user should be able to add a favorite and search through favorites', () => {
```
Comment (Ant-Shell) [inline cypress/e2e/favorites-spec.cy.js]: We should remove the 'only' from line 70 in favorites-spec.cy.js

File: src/components/SingleResultPage/SingleResultPage.js [hunk 2]
```diff
@@ -52,20 +54,20 @@ const SingleResultPage = ({ business, user, addFavorite, deleteFavorite }) => {
   return(
     <section className="single-result-section">
```
Comment (Ant-Shell) [inline src/components/SingleResultPage/SingleResultPage.js:55]: We may need to make some adjustments to styling of `<section className="single-result-section">`, `<article className="single-result">` and/or `<div className="single-title-container">` - the some of the text lands outside of the container on larger screens

Screenshot 2022-12-14 at 9 37 51 AM

File: src/components/SingleResultPage/SingleResultPage.js [hunk 3]
```diff
@@ -52,20 +54,20 @@ const SingleResultPage = ({ business, user, addFavorite, deleteFavorite }) => {
...
-          <p>{price}</p>
-          <div className="single-rating-container">
-            <AiFillStar/>
-            <p>{rating} / 5</p>
+          <div className="single-title-container">
+            <h1 className="single-title title-and-rating">{title}</h1>
+            <AiFillStar className="single-rating title-and-rating star-space"/>
+            <p className="single-rating title-and-rating">{rating} / 5</p>
           </div>
+          <p className="single-price">Is it expensive? {price}</p>
```
Comment (Ant-Shell) [inline src/components/SingleResultPage/SingleResultPage.js]: Maybe change to this?

`<p className="single-price">Price rating: {price}</p>`

File: cypress/e2e/favorites-spec.cy.js (hunk 1 above)
Comment (coleanthony1990) [reply cypress/e2e/favorites-spec.cy.js]: ah thankyou for looking at that. I can work on that today

**Comment by Ant-Shell** [comment]
Maybe we give the star icons for favoriting on the `ResultCard` a black border? Hard to see in some cases. Might be tricky since some images are darker though ...

Screenshot 2022-12-14 at 10 00 27 AM

File: cypress/e2e/single-results-spec.cy.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    cy.get('.input-email').type('example@email.com')
+    cy.get('.login-btn').click()
+    cy.get('[data-cy="search"]').type('denver')
+    cy.get('[data-cy="market-radio"]').click()
+    cy.intercept('GET', Cypress.env('market'), {
+      fixture: 'singleMarketData.json'
+    })
+    cy.get('[data-cy="submit"]').click()
+    cy.get('.business-card-image').click()
+    cy.get(['[data-cy="unfavorited]'])
```
Comment (vfields) [inline cypress/e2e/single-results-spec.cy.js]: If we are testing for whether or not this start exists, we might add a .should('exist') or a .should('be.visible') on line 55, and maybe there is a path where we are testing for it not existing/not being visible before logging in?

File: src/components/ResultCard/ResultCard.js [hunk 5]
```diff
@@ -18,6 +15,8 @@ const ResultCard = ({ title, photo, rating, id, alias, user, addFavorite, delete
       return acc;
     }, false)
   }
+
+  const workingImg = photo ? photo : eatLocalIcon
```
Comment (vfields) [inline src/components/ResultCard/ResultCard.js:19]: Love this!

**Review by vfields — COMMENTED** [review]
Happy to help with the cypress or the styling! Thank you for all of your work on this, Cole!

**Review by vfields — APPROVED** [review]
Reviewed refactoring with the team!

**Review by Ant-Shell — APPROVED** [review]
Reviewed with the team - approved!

**Pull request merged**